appcues/src/main/java/com/appcues/data/remote/DataRemoteKoin.kt

```diff
@@ -1,30 +1,63 @@
 package com.appcues.data.remote
 
 import com.appcues.AppcuesConfig
-import com.appcues.data.remote.retrofit.AppcuesService
-import com.appcues.data.remote.retrofit.RetrofitAppcuesRemoteSource
-import com.appcues.data.remote.retrofit.RetrofitWrapper
+import com.appcues.data.remote.appcues.AppcuesRemoteSource
+import com.appcues.data.remote.appcues.AppcuesService
+import com.appcues.data.remote.appcues.MetricsInterceptor
+import com.appcues.data.remote.customerapi.CustomerApiHostInterceptor
+import com.appcues.data.remote.customerapi.CustomerApiRemoteSource
+import com.appcues.data.remote.customerapi.CustomerApiService
+import com.appcues.data.remote.imageupload.ImageUploadRemoteSource
+import com.appcues.data.remote.imageupload.ImageUploadService
+import com.appcues.data.remote.sdksettings.SdkSettingsRemoteSource
+import com.appcues.data.remote.sdksettings.SdkSettingsService
 import com.appcues.di.KoinScopePlugin
 import okhttp3.HttpUrl.Companion.toHttpUrl
 import org.koin.dsl.ScopeDSL
+import java.util.concurrent.TimeUnit.SECONDS
 
 internal object DataRemoteKoin : KoinScopePlugin {
 
-    private const val BASE_URL = "https://api.appcues.net/"
-
     override fun ScopeDSL.install() {
-        scoped<AppcuesRemoteSource> {
+        scoped {
             val config: AppcuesConfig = get()
-            RetrofitAppcuesRemoteSource(
-                appcuesService = getAppcuesService(config.apiBasePath ?: BASE_URL),
-                accountId = config.accountId,
+            AppcuesRemoteSource(
+                service = RetrofitWrapper(
+                    baseUrl = (config.apiBasePath ?: AppcuesRemoteSource.BASE_URL).toHttpUrl(),
+                    interceptors = listOf(MetricsInterceptor()),
```

Comment:
this is the only one that needs the MetricsInterceptor at this point - we can now configure interceptors on a per service basis.